src/Integration.Vsix/CFamily/ProcessRunner.cs

```diff
@@ -116,66 +120,87 @@ public bool Execute(ProcessRunnerArguments runnerArgs)
                     else
                     {
                         LogMessage(CFamilyStrings.MSG_ExecutionCancelled);
-                        return false;
+                        return;
                     }
                 }
 
-                var result = WaitForProcessToFinish(process, runnerArgs);
-                return result;
+                using (var timeoutTask = Task.Factory.StartNew(() => WaitForProcessToFinish(process, runnerArgs)))
```

Comment:
What should I use instead of StartNew? :/ 
I can't use `await TaskScheduler.Default` since we're already on a background thread :/

Reply:
So we're tying up two background threads for each analysis? That's not great.
Did you look at the threading docs for VS? https://github.com/microsoft/vs-threading/blob/master/doc/index.md

Reply:
Yes and I'm still not sure how to handle it instead of new Task :/